specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/examples/virtualMachineScaleSetExamples/VirtualMachineScaleSet_Create_WithAutomaticRepairs.json

```diff
@@ -134,6 +147,19 @@
           "upgradePolicy": {
             "mode": "Manual"
           },
+		  "scheduledEventsPolicy": { 
+			"scheduledEventsAdditionalPublishingTargets": { 
```

Comment:
scheduledEventsAdditionalPublishingTargets should be indented to the right. 
Please run prettier on these files. 
https://levelup.gitconnected.com/improve-code-quality-with-the-vscode-prettier-formatter-d43e8d329505?gi=c7f438e2a616#:~:text=To%20use%20Prettier%2C%20you%20need,clicking%20and%20selecting%20Format%20Document%20.